src/appendbuf.rs

```diff
@@ -0,0 +1,460 @@
+use std::fmt::{self, Show, Formatter};
+use std::mem;
+use std::num::Int;
+use std::sync::Arc;
+
+use raw::{Allocator, RawIobuf};
+use iobuf::{Iobuf};
+use impls::{AROIobuf};
+
+/// ## AppendBuf
```

Comment:
This doesn't match the style of the other impls. The first line should be "Append-only input buffer"